include/deal.II/base/geometry_info.h

```diff
@@ -2312,18 +2312,35 @@ struct GeometryInfo
                              const bool         face_flip        = false,
                              const bool         face_rotation    = false);
 
+  /**
+   * Map the vertex index @p vertex of a line in standard orientation to one of a
+   * face with arbitrary @p line_orientation. The value of this flag default to
+   * <tt>true</tt>.
+   */
   static unsigned int
-  standard_to_real_line_vertex(const unsigned int line,
-                               const bool         face_orientation = true);
+  standard_to_real_line_vertex(const unsigned int vertex,
+                               const bool         line_orientation = true);
 
+  /**
+   * Decompose the vertex index in a quad into a pair of a line index and a
+   * vertex index within this line.
+   */
   static std::array<unsigned int, 2>
-  standard_corner_to_line_vertex_index(const unsigned int corner);
+  standard_quad_vertex_to_line_vertex_index(const unsigned int vertex);
```

Comment:
Isn't this non-unique? That is, aren't there multiple lines that contain the vertex?

Reply:
Indeed. The function selects one possible one as it used to be done in `CellAccessor`. This is generalizable to TETs (see: https://github.com/peterrum/dealii/pull/7#issuecomment-630276686).

Reply:
Can you update the documentation to say that the function chooses one, and maybe which one (or say that it chooses "one" and that it is purposefully unspecified which one that is)?

Reply:
I have more or less copied the comment from `CellId`. Is that fine this way?